Gemfile

```diff
@@ -1,4 +1,4 @@
-ruby '~> 2.4'
+ruby '>= 2.4'
```

Comment:
Not sure if we want to keep this `ruby '~> 2.4'` but if we do I think we need `>=` to support Ruby 2 and 3.
<img width="1097" alt="Screen Shot 2021-01-10 at 18 06 59" src="https://user-images.githubusercontent.com/1557529/104118767-b5690a00-536e-11eb-918c-74b4a11b99dd.png">